test/load/packer/loadtest-ami/scripts/loadtest.js

```diff
@@ -0,0 +1,41 @@
+import http from 'k6/http';
```

Comment:
@aaron-lane can you take a looksy at this?

Reply:
Looks pretty good! You may want  to add [checks](https://k6.io/docs/using-k6/checks) for each of the HTTP PUT requests to verify that the requests are successful. If one request is required to succeed in order for the next to succeed then you can also use [fail](https://k6.io/docs/javascript-api/k6/fail-err) in some conditional logic.